.golangci.reference.yml

```diff
@@ -2734,13 +2681,22 @@ linters:
     - unparam
     - unused
     - usestdlibvars
-    - varcheck
     - varnamelen
     - wastedassign
     - whitespace
     - wrapcheck
     - wsl
     - zerologlint
+    - deadcode # Deprecated
+    - exhaustivestruct # Deprecated
+    - golint # Deprecated
+    - ifshort # Deprecated
+    - interfacer # Deprecated
```

Comment:
It seems that `ifshort` and `interfacer` have no alternatives from other linters. Perhaps we should fork their repositories under the https://github.com/golangci organization?

Reply:
1. The fork is something that we try to avoid.
2. they are deprecated for multiple years.

From interfacer readme:

> Deprecated: A tool that suggests interfaces is prone to bad suggestions, so its usefulness in real code is limited.

For `ifshort`, it's the decision of the author to not transfer it.
https://github.com/golangci/golangci-lint/pull/3034#issuecomment-1200717412

Reply:
😢